hedera-mirror-rest/utils.js

```diff
@@ -983,11 +1002,55 @@ const loadPgRange = () => {
   pgRange.install(pg);
 };
 
+/**
+ * Checks that the timestamp filters contains either a valid range (greater than and less than filters that do not span beyond
+ * a configured limit), or at least one equals operator.
+ *
+ * @param timestampFilters an array of timestamp filters
+ */
+const maxTimestampRange = parseDuration(config.maxTimestampRange);
+const checkTimestampRange = (timestampFilters) => {
+  let latest = undefined;
```

Comment:
nit

```suggestion
  let latest;
```
initialize it to `undefined` is the same as uninitialized

Reply:
Fixed